src/researchPage.js

```diff
@@ -1,11 +1,80 @@
-import { Wrapper, H1 } from "./components/styled";
+import { Wrapper , ExternalLink } from "./components/styled";
 import { Link } from "react-router-dom";
 import Button from "./components/button/button";
+import Header from "./components/header/header";
+import ColorBlock from "./components/colorBlock/colorBlock";
+import { BlockContainer } from "./components/colorBlock/colorBlock.styles";
 
 function ResearchPage() {
   return (
     <Wrapper>
-      <H1 bold>this is the research</H1>
+      <Header
+        text={"Color Theory"}
+        subtext={
+          "Learn more about the meaning behind each color palette and our research sources."
+        }
+      />
+      <BlockContainer>
+      <ColorBlock
+        emotion={"relax, de-stress, calm, attention"}
+        color1={"#D9E5F3"}
+        color2={"#B2D3ED"}
+        color3={"#B2B5D9"}
+        color4={"#296CA0"}
+      />
+      <ColorBlock
+        emotion={"creativity, inspiration"}
+        color1={"#FDE8C0"}
+        color2={"#CFC8F8"}
+        color3={"#FCC663"}
+        color4={"#70669D"}
+      />
+      <ColorBlock
+        emotion={"healing, pain relief, comfort"}
+        color1={"#E7F9C3"}
+        color2={"#ACDCD1"}
+        color3={"#97CD80"}
+        color4={"#719680"}
+      />
+      <ColorBlock      
+      emotion={"strength, courage, focus"}
+        color1={"#FFA5A4"}
+        color2={"#D78F8E"}
+        color3={"#BABABA"}
+        color4={"#565656"}
+        /> 
+      <ColorBlock
+        emotion={"joy, energy, positivity"}
+        color1={"#FFFFA2"}
+        color2={"#F6E7A4"}
+        color3={"#FEBB3D"}
+        color4={"#D46C00"}
+      />
+      </BlockContainer>
+      <p>Sources</p>
+        <ExternalLink target="_blank" href={ 
+            "http://www.arttherapyblog.com/online/color-psychology-psychologica-effects-of-colors/#.YYLxU73MK58."}>
+          {" "}
```

Comment:
what are these `{" "}` spaces here for? remove them on all of these links if they're not doing anything